back-end/services/projects.service.js

```diff
@@ -1,55 +1,112 @@
 import ProjectsDAO from "../dao/projectsDAO.js";
 
+
 export default class ProjectsService {
-  static async createProject(body) {
-    try {
-        console.log(body);
-        await ProjectsDAO.createProject(body);
-        return { status: "success" };
-    } catch (e) {
-        return { status: "failure" };
+    /**
+     * Creates a new project and returns the status.
+     * @param body
+     * @returns {Promise<{status: string}>}
+     */
+    static async createProject(body) {
```

Comment:
Again, thank you for the documentation! 

As a reminder, since the "static" keyword seems unnecessary, we should discuss as a team again on how we'll refactor to get rid of it.